examples/wildfly-core-channel.yaml

```diff
@@ -0,0 +1,10 @@
+---
+schemaVersion: "2.0.0"
+repositories:
+  - id: "central"
+    url: "https://repo1.maven.org/maven2/"
+  - id: "jboss-public"
+    url: "https://repository.jboss.org/nexus/content/groups/public/"
+manifest:
+  url: "file:///home/thofman/Projects/prospero/examples/wildfly-core-manifest.yaml"
```

Comment:
I guess the github url will be in the final PR?

Reply:
Yes, I think the URL to GitHub are best for this use cases because they are absolute and will stay active. Ideally it should also be pointing to the "main" branch.

Reply:
Updated to GH urls.